MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -32,6 +38,7 @@ final class FeedPresenter {
 	}
 	
 	func didFinishLoadingFeed(with error: Error) {
+		errorView.display(FeedErrorViewModel(error: error))
```

Comment:
As described in my previous comment, the FeedErrorViewModel should hold an `Optional` formatted error string (not an `Error` type).

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `errorView.display(FeedErrorViewModel(error: .none))` to hide the error.

And here in `didFinishLoadingFeed(with error: Error)`, it should provide the view with a string error message:

```suggestion
		errorView.display(FeedErrorViewModel(error: Localized.Feed.loadError))
```

This way, the ViewController doesn't need to decide when to show/hide the error and which error to show. It's easier to manage presentation state transitions when all changes are centralized in the presenter.

Reply:
fixed